### PR TITLE
dockerfile for dev added

### PR DESCRIPTION
**Pull request by NaveenShanmugavel18**

### CODE REVIEW

File: meme-text-service/Dockerfile.dev [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+RUN apt-get update -y && apt-get -y install tesseract-ocr tesseract-ocr-tam python3 python3-pip
+
+WORKDIR /usr/src/app
+
+COPY requirements.txt ./
+
+RUN pip3 install -r requirements.txt
+
+COPY . .
```
Comment (adithyaselv) [inline meme-text-service/Dockerfile.dev:11]: line 7 can be replaced with this ?

**Comment by adithyaselv** [comment]
looks good but should we add a compose file which also mounts the app folder to local to enable development using IDE ?

File: meme-text-service/Dockerfile.dev (hunk 1 above)
Comment (NaveenShanmugavel18) [reply meme-text-service/Dockerfile.dev:11]: Went through the docker caching layer. Looks like whether python or node will install dependencies only if package.json or req.txt changes.

That is the reason behind line 7

**Comment by NaveenShanmugavel18** [comment]
@adithyaselv yea we can. After we dockerize all services in meiyaluma. Good idea

**Review by adithyaselv — APPROVED** [review]
Great work

**Pull request merged**